bazaarvoice/emodb
# postman tests for delete role emodb endpoint

## What Are We Doing Here?

Postman tests have been created for DELETE /uac/1/role/{group}/{id} endpoint
## How to Test and Verify

1. Run this branch on Jenkins 
2. Verify that tests for DELETE /uac/1/role/{group}/{id} endpoint have been executed and past

## Risk
broken or unstable tests have been added to test suite

### Level 
`Low`

### Required Testing

`Smoke`
### Risk Summary
Existing functionality has not been affected, but new unstable tests could be introduced to test scope.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
